ec/src/models/short_weierstrass_jacobian.rs

```diff
@@ -293,13 +293,13 @@ impl<P: Parameters> PartialEq for GroupProjective<P> {
         // The points (X, Y, Z) and (X', Y', Z')
         // are equal when (X * Z^2) = (X' * Z'^2)
         // and (Y * Z^3) = (Y' * Z'^3).
-        let z1 = self.z.square();
-        let z2 = other.z.square();
+        let z1z1 = self.z.square();
```

Comment:
Drive by change to make this more consistent with the rest of the SW arithmetic code